### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by ghost** [react-hook-form/react-hook-form#4060] [Show and tell]
I don't know how else to reach the developers of this just to say 'thank you'. I have been struggling with Formik for some time. I just don't find it that easy to use so I wanted to try something else. I found React Hook Form and it has blown me away with how easy it is to use. I think it is brilliant and I just wanted to say thank you to all the people involved with this project. I no longer hate building React forms thanks to you!

**Comment by bluebill1049** [comment]
Thank you @Jonofat! That's so kind of you to write such a lovely message. We are constantly improving the lib to make everyone's life easier. ❤️
